mindjunkies/accounts/tests/test_view.py

```
@@ -0,0 +1,44 @@
+import pytest
+from django.urls import reverse
+from mindjunkies.accounts.models import User
```

Ruff (0.8.2)

`mindjunkies.accounts.models.User` imported but unused

Remove unused import: `mindjunkies.accounts.models.User`

(F401)